presto-native-execution/presto_cpp/main/common/Counters.cpp

```diff
@@ -151,6 +151,20 @@ void registerPrestoCppCounters() {
       kCounterSsdCacheCumulativeCachedBytes, facebook::velox::StatType::AVG);
   REPORT_ADD_STAT_EXPORT_TYPE(
       kCounterSsdCacheCachedBytes, facebook::velox::StatType::AVG);
+  REPORT_ADD_STAT_EXPORT_TYPE(
+      kCounterHiveFileHandleCacheNumElementsFormat,
+      facebook::velox::StatType::AVG);
```

Comment:
Is it even going to work: register format string here and report formatted string above?

Reply:
Not going to work. Hmmm I guess it cannot be dynamic after all.

Reply:
Ok one way of doing so is moving the exporting to PeriodicTaskManager when it's used. That way we can do dynamic

Reply:
Yes, we can register/declare/export when we know them already, the formatting is not needed.